### ISSUE TITLE
Nixpkgs fails to build on unstable

### ISSUE THREAD

**Issue by JCapucho** [zhaofengli/colmena#50]
Hi, I'm using `colmena` with a flake to build a node that uses the `nixos-unstable` branch of `nixpkgs` (commit is https://github.com/NixOS/nixpkgs/commit/689b76bcf36055afdeb2e9852f5ecdd2bf483f87), but it's failing to build.

In the start of the log I see a warning with
```
capucho-nixos | trace: warning: The following Nixpkgs configuration keys set in meta.nixpkgs will be ignored: path
```
Which leads me to believe that the issue might be caused by ignoring the `path` attribute introduced in https://github.com/NixOS/nixpkgs/pull/153594

Here's the log of running `colmena build --show-trace --verbose`
[log.txt](https://github.com/zhaofengli/colmena/files/7934091/log.txt)

**Comment by zhaofengli** [comment]
From the logs, it seems like it's a problem with the split docs build:

```
capucho-nixos | Cacheable portion of option doc build failed.
capucho-nixos | Usually this means that an option attribute that ends up in documentation (eg `default` or `description`) depends on the restricted module arguments `config` or `pkgs`.
```

Can you confirm whether e58dde1be08cf7fbac7299e928b6d0e143edbec9 fixes the issue?

**Comment by JCapucho** [comment]
Just tested on the latest commit, it's all working now, thanks for the quick fix

**Comment by zhaofengli** [comment]
Let's keep this open until I also backport the fix to 0.2.

**Comment by zhaofengli** [comment]
Closing since 0.2.1 has landed in both `nixos-unstable` and `nixos-21.11`. Thanks for the report!
